Rubberduck.VBEEditor/QualifiedModuleName.cs

```diff
@@ -60,6 +60,34 @@ public QualifiedModuleName(VBComponent component)
         }
 
         /// <summary>
+        /// Creates a QualifiedModuleName for removing renamed declarations.
+        /// Do not use this overload.
+        /// </summary>
+        public QualifiedModuleName(VBComponent component, string oldComponentName)
```

Comment:
No.  `QualifiedModuleName(VBProject project)` leaves the component name null.  `public QualifiedModuleName(VBComponent component)` gets the name from the component, which would be the new name by this point.  `QualifiedModuleName(string projectName, string projectPath, string componentName)` does not calculate the correct project id.